admin/menues.rb

```diff
@@ -1,37 +1,37 @@
 ActiveAdmin.register Goldencobra::Menue, as: "Menue" do
-  menu parent: I18n.t('active_admin.menues.parent'), label: I18n.t('active_admin.menues.as'), if: proc{can?(:update, Goldencobra::Menue)}
+  menu parent: I18n.t("active_admin.menues.parent"), label: I18n.t("active_admin.menues.as"), if: proc{can?(:update, Goldencobra::Menue)}
```

Comment:
My aim was to get one step further on styling/refactoring. Did not want to refactor multiple things at once because of the complexity of testing afterwards.